brownfield_django/main/views.py

```diff
@@ -297,6 +291,7 @@ def send_student_email(self, student):
 
     def post(self, request, pk):
         '''This is really really ugly as is get method need to clean up.'''
+        print request.POST
```

Comment:
mod_wsgi does not like print statements. (http://blog.dscpl.com.au/2009/04/wsgi-and-printing-to-standard-output.html)

This needs to be removed before a merge.